Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/broadinstitute/viral-baseimage:0.2.1
+FROM quay.io/broadinstitute/viral-baseimage:ct-bump-conda-python
```

Comment:
this should update to a numbered version

Reply:
👍 Thanks, yeah, that's left over from local testing.